docker-compose/docker-compose.yml

```
@@ -0,0 +1,413 @@
+      - redpanda_network
+    depends_on:
+      - minio
+    ports:
+      - 8181:8181
+    environment:
+      - AWS_ACCESS_KEY_ID=minio
+      - AWS_SECRET_ACCESS_KEY=redpandaTieredStorage7
+      - AWS_REGION=local
+      - CATALOG_WAREHOUSE=s3://redpanda/
+      - CATALOG_IO__IMPL=org.apache.iceberg.aws.s3.S3FileIO
+      - CATALOG_S3_ENDPOINT=http://minio:9000
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)